### PR TITLE
Fixes and instructions for Todolist + log4shell on K8s

### PR DESCRIPTION
**Pull request by ericsmalling**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,47 @@ It's divided into modules, each one having its own README:
 
 * [Todolist Goof](todolist-goof/README.md)
 * [Log4Shell Goof](log4shell-goof/README.md)
+
+---
+## Kubernetes based Todolist + Log4Shell exploit
```
Comment (dogeared) [inline README.md:11]: @ericsmalling - should this be in the README within the todolist-goof folder?

File: todolist-goof/Dockerfile [hunk 2]
```diff
@@ -12,3 +12,11 @@ COPY --chown=tomcat:tomcat web.xml /usr/local/tomcat/conf/web.xml
 COPY --from=build /usr/src/goof/todolist-web-struts/target/todolist /usr/local/tomcat/webapps/todolist
 COPY --from=build /usr/local/openjdk-8/bin/native2ascii /docker-java-home/jre/bin/native2ascii
 COPY --from=build /usr/local/openjdk-8/lib/tools.jar /docker-java-home/jre/lib/tools.jar
+
+
```
Comment (dogeared) [inline todolist-goof/Dockerfile:16]: Are all the newlines necessary?

**Review by dogeared — APPROVED** [review]
If it makes sense to have the new content in README at the root, leave it there. Otherwise, move to the todolist-goof/README.md. Either way - :shipit:

File: README.md (hunk 1 above)
Comment (ericsmalling) [reply README.md:11]: It straddles both - it's a demo of todolist + log4shell

Comment (ericsmalling) [reply README.md:11]: I could move it into it's own README though

File: todolist-goof/Dockerfile (hunk 2 above)
Comment (ericsmalling) [reply todolist-goof/Dockerfile:16]: I put them in so it doesn't show up w/out scrolling when showing to the audience when it's not relevant.

File: README.md (hunk 1 above)
Comment (ericsmalling) [reply README.md]: I did that move in in latest commit

**Comment by dogeared** [comment]
LGTM!

**Pull request merged**